common/nym_offline_compact_ecash/Cargo.toml

```diff
@@ -20,6 +20,7 @@ rand = { workspace = true }
 thiserror = { workspace = true }
 sha2 = "0.9"
 bs58 = { workspace = true }
+utoipa = { workspace = true }
```

Comment:
I thought you removed all utoipa stuff from the ecash library?

Reply:
removed